openqabot/approver.py

```diff
@@ -131,6 +132,52 @@ def is_job_marked_acceptable_for_incident(self, job_id: int, inc: int) -> bool:
             pass
         return False
 
+    @lru_cache(maxsize=512)
+    def was_ok_before(self, job_id: int, inc: int) -> bool:
+        log.info("Current aggregate test %s failed. Will look at older jobs" % job_id)
```

Comment:
Depending on how many such log messages appear in a usual run consider to make that debug. Also I would phrase it

```suggestion
        log.info("Current aggregate test %s failed. Looking up job history" % job_id)
```

Reply:
will remove this and rethink logging for this thing, reducing duplicated informations (if later on there is a log for each case (job found and ignore failure, job not found and reason) there is no need of this log announcing the start of this search